spectroperf.go

```diff
@@ -100,7 +100,7 @@ func main() {
 			Username: config.Username,
 			Password: config.Password,
 		},
-		SecurityConfig: gocb.SecurityConfig{TLSSkipVerify: config.TlsSkipVerify},
+		SecurityConfig: gocb.SecurityConfig{TLSSkipVerify: config.TlsSkipVerify, TLSRootCAs: caCertPool},
```

Comment:
seems unrelated to this change.  needs rebase or other commits need to go in first?

Reply:
Yes this is part of the change in the other PR.